archive.php

```diff
@@ -0,0 +1,36 @@
+<?php
+	include 'navbar.php';
+?>
+
+<html>
+<head>
+<link rel="stylesheet" href="http://maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
+<link rel="stylesheet" href="archive.css">
+</head>
+<body>
+	<div id="container">
+		<script>
+		var youtube_videos = ['<iframe width="560" height="315" src="https://www.youtube.com/embed/15tLVw0V-bM" frameborder="0" allowfullscreen></iframe>',
```

Comment:
@hitesh4 

Instead of putting code with iframe in here, just put the youtube video link here. And generate the iframe code dynamically in the Javascript.